main/main_timer_sync.cpp

```diff
@@ -29,6 +29,8 @@
 /*************************************************************************/
 
 #include "main_timer_sync.h"
+#include "core/math/math_funcs.h"
+#include "core/os/os.h"
```

Comment:
Style nitpick but I like to keep one line between the `.cpp` file's main header and the other engine headers:
```suggestion
#include "main_timer_sync.h"

#include "core/math/math_funcs.h"
#include "core/os/os.h"
```